### PR TITLE
Add source code URI to the Ruby gemspec file

### PR DESCRIPTION
**Pull request by jmonteiro**
As a developer, I would like to easily find the source code origin for this gem when browsing through https://rubygems.org/gems/google-protobuf.

In order to solve this, I've followed instructions at https://guides.rubygems.org/specification-reference/#metadata by adding the source_code_uri metadata key pair.

# Impact

This will cause https://rubygems.org/gems/google-protobuf to show the link in the following place:

nokogiri___RubyGems_org___your_community_gem_host

### CODE REVIEW

**Comment by jmonteiro** [comment]
@googlebot I signed it!

File: ruby/google-protobuf.gemspec [hunk 1]
```diff
@@ -7,6 +7,7 @@ Gem::Specification.new do |s|
   s.homepage    = "https://developers.google.com/protocol-buffers"
   s.authors     = ["Protobuf Authors"]
   s.email       = "protobuf@googlegroups.com"
+  s.metadata    = { "source_code_uri" => "https://github.com/protocolbuffers/protobuf/tree/master/ruby" }
```
Comment (TeBoring) [inline ruby/google-protobuf.gemspec]: Can you replace the master with something related to s.version?

**Comment by TeBoring** [comment]
Any update?

**Comment by jmonteiro** [comment]
@TeBoring added a dynamic change to it, let me know if you think it's better to write it manually (as in a `git_tag = "v3.9.0-rc1"`).

**Comment by TeBoring** [comment]
LG now

**Pull request merged**